client/src/components/reader/Annotation/Editor.js

```diff
@@ -45,41 +38,29 @@ export default class AnnotationSelectionEditor extends PureComponent {
 
   handleSubmit = event => {
     event.preventDefault();
-    const { subject, startNode, startChar, endNode, endChar } = this.props;
-    const { body, isPrivate } = this.state;
-    const annotation = {
-      subject,
-      startNode,
-      startChar,
-      endNode,
-      endChar,
-      body,
-      // eslint-disable-next-line quote-props
-      private: isPrivate,
```

Comment:
This line is causing the private annotation regression.  State key has to be changed to `private` from `isPrivate` or that attribute need to be assigned.